scheduler/src/test/java/org/cloudfoundry/autoscaler/scheduler/util/TestConfiguration.java

```diff
@@ -4,6 +4,6 @@
 
 @TestPropertySource(properties = { "scalingenginejob.reschedule.interval.millisecond=100",
 		"scalingenginejob.reschedule.maxcount=5", "autoscaler.scalingengine.url=http://localhost:8090",
-		"scalingengine.notification.reschedule.maxcount=2" })
+		"scalingengine.notification.reschedule.maxcount=2", "server.ssl.enabled=false" })
```

Comment:
No we donot, specified server.ssl.enabled=false property because by specifying server.ssl properties in application.properties, tomcat goes looking for the files  and junit test cases fail if keystore file etc are not found. Also embedded tomcat looks for them in temp directory if relative path is given.